package.json

```diff
@@ -17,52 +17,53 @@
   },
   "scripts": {
     "build": "ember build",
+    "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
     "test": "ember test",
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.14.1"
+    "ember-cli-babel": "^7.7.3"
   },
   "devDependencies": {
-    "@ember-decorators/babel-transforms": "^2.0.1",
-    "broccoli-asset-rev": "^2.7.0",
-    "ember-cli": "~3.1.3",
-    "ember-cli-addon-docs": "elwayman02/ember-cli-addon-docs#hero-yield",
+    "@ember/optional-features": "^0.7.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-cli": "~3.12.0",
+    "ember-cli-addon-docs": "^0.6.14",
```

Comment:
Unfortunately, that fork doesn't work with the latest Ember, so, I had to update it to latest ember-cli-addon-docs.

I looked at the dummy app and fixed the issues with it.
I just removed the multi-yield and the Ember logo, because the word "Ember" is in the hero already with the latest addon version.